TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label profileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
```

Comment:
where is your prefix? I'm not sure why you keep having this mistake.

Reply:
I'm sorry. My misstake. I will fix it

Reply:
Please check again. I am sorry